_data/identity.yml

```diff
@@ -158,6 +158,15 @@ websites:
           text: "2FA is only available on paid plans."
       doc: https://www.securesafe.com/en/faq/general-questions/how-can-i-activate-2-factor-authentication/
 
+    - name: The Identity Hub
+      url: https://www.theidentityhub.com
+      img: theidentityhub.png
+      tfa: Yes
+      sms: Yes
```

Comment:
I don't see anything on the documentation page you provided about SMS authentication. Are you sure you support it?

Reply:
Only one link to documentation was possible. Here https://www.theidentityhub.com/hub/Documentation/#ConfigureSmsProvider is the explaination on how to configure a SMS provider that will allow SMS authentication.